addons/payment_stripe/controllers/main.py

```diff
@@ -171,3 +169,40 @@ def _verify_webhook_signature(self, webhook_secret):
             return False
 
         return True
+
+    # -------------------------------------------
+    # Stripe Connect Onboarding
+    # -------------------------------------------
+    @http.route('/payment/stripe/onboarding/return/<int:acquirer_id>', type='http', auth='public', csrf=False)
+    def stripe_onboarding_return(self, acquirer_id, **kwargs):
+        """Stripe return URL
+
+            This route is used by Stripe to return after or during the onboarding.
+            Once called, the controller tests the account status of the Onboarded Stripe account
+            and redirect to it.
+
+            :returns: Redirection to Stripe Acquirer Form
+        """
+        onboarding = kwargs.get('onboarding', 'payment')
+        menu = False
+        if onboarding == 'website_sale':
+            menu = request.env.ref('website_sale.menu_ecommerce_payment_acquirers', raise_if_not_found=False)
+        elif onboarding == 'sale':
+            menu = request.env.ref('sale.sale_menu_root', raise_if_not_found=False)
+        return request.redirect(
+            '/web?#action=payment_stripe.action_payment_acquirer_onboarding&id=%s&menu_id=%s'
+            % (acquirer_id, menu and menu.id)
+        )
```

Comment:
If Stripe redirects the user with a POST request, I'm a bit worried that they will be prompted to log in after being redirected to the acquirer form because their session was dropped in this controller. Could you please make some testing to see if that happens when the session cookie is cleared in this controller?

More information here: https://github.com/odoo/odoo/blob/master/addons/payment_adyen/controllers/main.py#L192-L198

Reply:
I tested on Firefox that doesn't implement the default `SameSite=Lax` behavior and it seems I don't lose the session.

Yet, I don't know if I understood correctly what you said so it may still be interesting to review it together.

Reply:
Since Stripe does only a 302 with our route as a `Location`, I guess it has no effect and the session cookie is taken into account in any case.

I think it has no effect here but I would surely accept to add it in case of any doubt you would have.

Reply:
It looks like most browsers redirect with POST even if the request was originally a GET so let's forget about it for now. In the worst case, it's super easy to fix ;)